### PR TITLE
Updated embeddable video documentation.

### PR DESCRIPTION
**Pull request by Sam152**

### CODE REVIEW

File: modules/media_entity/embeddable_video.md [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
 
-- YouTube
+- YouTube and Vimeo (packaged with the module)
+- Facebook
+- Vine
+- Dailymotion
+- Instagram
+- Ted
 - AOL
-- Grab networks
```
Comment (slashrsm) [inline modules/media_entity/embeddable_video.md:13]: We should add Grab networks support to VEF to have feature parity.

Comment (Sam152) [inline modules/media_entity/embeddable_video.md:13]: Agreed.

Comment (slashrsm) [inline modules/media_entity/embeddable_video.md:13]: Let's create a follow-up then.

**Comment by slashrsm** [comment]
Merged. Thanks!

**Pull request merged**